ClearKeep/Scenes/ChatGroup/Views/CommonUI/ListUser.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ListUser.swift
+//  ClearKeep
+//
+//  Created by đông on 30/03/2022.
+//
+
+import SwiftUI
+import Common
+import CommonUI
+
+private enum Constant {
+	static let sizeImage = 64.0
+}
+
+struct ListUser: View {
+	@State private(set) var isSelectedUser: Bool = false
+
+	var body: some View {
+		HStack {
+			ZStack {
+				AppTheme.shared.imageSet.splashLogo
+				AppTheme.shared.imageSet.userIcon
+			}
+			.frame(width: Constant.sizeImage, height: Constant.sizeImage)
+			.clipShape(Circle())
+			//			.background(Color.green)
+			Text("Alissa Baker".localized)
+				.frame(minWidth: 0, maxWidth: .infinity, alignment: .leading)
+			CheckBoxButtons(text: "", isChecked: $isSelectedUser)
+		}
+	}
+}
+
+struct ListUser_Previews: PreviewProvider {
+    static var previews: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/9578f885ffed383bd6377ad58df523645b5bb3fe/ClearKeep/Scenes/ChatGroup/Views/CommonUI/ListUser.swift#L36"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>ListUser.swift:36</code></td>
    </tr>
  </tbody>
</table>